docs/1-Technical-Setup.md

```diff
@@ -96,38 +95,23 @@ colima stop
 >
 > please reference this
 > [help article](https://github.com/abiosoft/colima/blob/main/docs/FAQ.md#cannot-connect-to-the-docker-daemon-at-unixvarrundockersock-is-the-docker-daemon-running).
->
-> ---
 
-> ---
->
+
+> [!TIP]
 > If you run into an error such as:
->
+> 
 > `docker: 'compose' is not a docker command.`
+> 
+> try the following:
 >
-> add the following JSON to your `~/.docker/config.json file.`
-> ```json
-> "cliPluginsExtraDirs": [
->     "/opt/homebrew/lib/docker/cli-plugins"
-> ]
-> ```
->
-> ---
-
-### DBeaver (or a different database viewer)
-
-DBeaver is a free database viewer that we can use to interact with our database using a GUI. There
-are other tools that can be used instead, but DBeaver is a free option that we can install with
-homebrew.
-
-DBeaver can be installed by running the following commands:
+> - Create a symlink to the docker-compose plugin by running the following command:
+>  ```bash
+>  [ ! -d "~/.docker/cli-plugins" ] && mkdir -p "~/.docker/cli-plugins" && ln -s /opt/homebrew/opt/docker-compose/bin/docker-compose ~/.docker/cli-plugins/docker-compose
+>  ```
 
-```bash
-# install DBeaver with homebrew
-brew install --cask dbeaver-community
-```
+### NoSQL Workbench for DynamoDB 
 
-> if you have issues with DBeaver, another good option you can try is [Beekeeper Studio](https://www.beekeeperstudio.io/)
+NoSQL Workbench for DynamoDB is an AWS tool for designing and managing DynamoDB tables. You can install it by following the instructions [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/workbench.settingup.html).
```

Comment:
I tried running the prereq stuff in the NoSQL part (sudo add-apt-repository universe && sudo apt install libfuse2) and got these messages : 
sudo add-apt-repository universe && sudo apt install libfuse2
Password:
zchaloupecky is not in the sudoers file.  This incident will be reported.
 
IT ended up contacting me about these issues as I tried to update the sudoers file. I'm wondering if I even need these prereqs tho

Reply:
I updated the instructions to reflect that we should manually download the installer.